packages/go/openapi/src/responses/related-entity-query-results.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2024 Specter Ops, Inc.
+#
+# Licensed under the Apache License, Version 2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  **OK**
+
+  This endpoint returns a polymorphic response dependent upon which return type is requested
+  by the `type` parameter.
+content:
+  application/json:
+    schema:
+      oneOf:
+        - allOf:
+            - $ref: './../schemas/model.bh-graph.graph.yaml'
+            - title: graph
+        - title: count
+          type: integer
+        - allOf:
+            - $ref: './../schemas/api.response.pagination.yaml'
+            - title: list
+              type: object
+              properties:
+                data:
+                  type: array
+                  items:
+                    title: paged-node-list-entry

```

Comment:
This is probably only used in this one spot, but maybe we should move it out to its own file just in case we do need to reference it again. It least makes it consistent that every model has a file.

Reply:
The response model for this one is pretty unique and this is the only place it's used. If you feel strongly about it I can move it, but if it's "meh", my preference would be to leave it

Reply:
My preference would be on moving it out for consistency, but I'm not going to make a big fuss over it. We can leave it as-is.